rootfs/standard/usr/share/mynode_apps/albyhub/scripts/uninstall_albyhub.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source /usr/share/mynode/mynode_device_info.sh
+source /usr/share/mynode/mynode_app_versions.sh
+
+echo "==================== UNINSTALLING APP ===================="
+
+# The app folder will be removed automatically after this script runs. You may not need to do anything here.
+
+# TODO: Perform special uninstallation steps here
+
+# if pulled docker used
```

Comment:
Can you uncomment these? The container should be removed in the uninstall step.

Reply:
cleaned install